sdk/iot/modelsrepository/review/iot-modelsrepository.api.md

```diff
@@ -0,0 +1,51 @@
+## API Report File for "@azure/iot-modelsrepository"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { CommonClientOptions } from '@azure/core-client';
+import { OperationOptions } from '@azure/core-client';
+
+// @public
+export type dependencyResolutionType = "disabled" | "enabled" | "tryFromExpanded";
+
+// @public
+export interface GetModelsOptions extends OperationOptions {
+    dependencyResolution?: dependencyResolutionType;
+}
+
+// @public
+export function getModelUri(dtmi: string, repositoryUri: string, expanded?: boolean): string;
```

Comment:
Sometimes we move optional properties to an optional object even if it's only one, in case the optional properties might grow later. Up to you!

Reply:
for now considering the very limited scope of this function, I'm inclined to leave it outside of an object. I don't anticipate we add more optional parameters to it.